tools/src/test/python/dlpx/virtualization/_internal/fake_plugin/direct/multiple_warnings.py

```diff
@@ -75,6 +75,11 @@ def unconfigure(repository, source_config, virtual_source):
     pass
 
 
+@vfiles.virtual.cleanup()
+def cleanup(repository, source_config, virtual_source):
```

Comment:
nit: May be add extra args to make sure this code flags a new warning due to the additions in plugin_importer.yaml? It is a good way to confirm new validations for cleanup method are working. If you plan to change this, also change test_plugin_importer.py - check test_mulitple_warnings for ref.